corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -84,7 +84,8 @@ def __init__(self, suite, module, detail_section_elements):
         self.module = module
         self.detail_section_elements = detail_section_elements
         if self.module.is_multi_select():
-            self.case_session_var = "selected_cases"
+            # the instance is dynamic and its ID matches the datum ID
+            self.case_session_var = SearchSelectedCasesInstanceXpath.id
```

Comment:
Is this a functional change or is it cleaning up the code to not use a hard-coded string?